index.html

```diff
@@ -94,7 +94,7 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distri
             <div>Connect data sources & sinks.</div>
           </div>
           <ul>
-            <li><a href="connect-jdbc-avro/kstreams.html">Create a stream from a JDBC source</a></li>
+            <li><a href="connect-jdbc-kstreams-rekey-specificavro/kstreams.html">JDBC to SpecificAvro stream adding key</a></li>
```

Comment:
I suggest leaving out the "adding key" part for now. Can address later if this becomes an important distinction from other Connect tutorials.
![image](https://user-images.githubusercontent.com/141130/75571776-d6ab5b00-5a27-11ea-827f-e31772988625.png)

Reply:
@colinhicks the next tutorial I'd like to write will be similar to this one, but the keying functionality will be via Connect/SMT versus Kafka Streams. If we leave off "adding key", it will be harder to differentiate the tutorials even though they are very different workflows.  WDYT?

Reply:
@ybyzek, that makes sense. The space constraints are tough, here. Kind of like a newspaper.

Do you think it's reasonable to use just enough distinguishing detail to frame the story of each tutorial? For example, maybe the next tutorial you described can have a headline (link text) like:
`Key a stream with Kafka Connect SMT` or
`Key a stream with transformations`

Reply:
@colinhicks  We can't just decrease the font like the classifieds in a newspaper? j/k :)

The suggestions above do leave off two details:

1. SpecificAvro: the next KT was going to be JSON, but the description can explain that .  And it may not be as high priority to mention in the titl
2. JDBC: that it's a JDBC source connector versus another source connector, but that's also not as high priority, it's a detail actually

But all in all, given the space constraints, I think you found the best solution, thank you!

Reply:
Haha, we could always link-farm ourselves with `kafka.craigslist.org` 😛